src/data/set/lattice.lean

```diff
@@ -2,14 +2,40 @@
 Copyright (c) 2014 Jeremy Avigad. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jeremy Avigad, Leonardo de Moura, Johannes Hölzl, Mario Carneiro
-
--- QUESTION: can make the first argument in ∀ x ∈ a, ... implicit?
 -/
-import order.complete_boolean_algebra
 import data.sigma.basic
+import order.complete_boolean_algebra
 import order.galois_connection
 import order.directed
 
+/-!
+# The set lattice
+
+This file provides usual set notation for unions and intersections, a `complete_lattice` instance
+  for `set α`, some more set constructions, and proves basic results combining them.
+
+## Main declarations
+
+* `set.lattice_set`: `set α` along with `(⊆)`, `(⋂)` and `(⋃)` is a `complete_lattice`,
```

Comment:
```suggestion
* `set.lattice_set`: `set α` is a `complete_lattice` with `< = ⊂`, `≤ = ⊆`, `⊓ = ∩`, `⊔ = ∪`,
  `⨅ = ⋂` and`⨆ = ⋃`.
```

Reply:
Although maybe most of that belongs in the docstring for `set.boolean_algebra`

Reply:
I think explaining it twice is fine.